firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHttpClient.java

```diff
@@ -242,7 +249,16 @@ private void setUpUrlConnection(
   }
 
   private String getFetchUrl(String projectNumber, String namespace) {
-    return String.format(FETCH_REGEX_URL, projectNumber, namespace);
+    String fetchRegexUrl = FETCH_REGEX_URL;
```

Comment:
Could be more direct to return early if emulatedServiceSettings is null and if not, construct and return the emulator url.

Reply:
Done!